src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -38,16 +40,17 @@ internal class DefaultCircuitFactory : CircuitFactory
             _logger = _loggerFactory.CreateLogger<DefaultCircuitFactory>();
```

Comment:
Is there a reason why we do all these nullchecks here? Can we simply get rid of them?

Reply:
just got rid of all the unnecessary nullchecks